### ISSUE TITLE
Add content for Django library

### ISSUE THREAD

**Issue by Einsteinia11** [thecuriousteam/PyLibLog#33]

**Comment by Ro-shni** [comment]
would you like me to add the first blog post about Getting Started with Django and its Installation

**Comment by Einsteinia11** [comment]
Hey,
Yes you can start working on it. Make sure you describe briefly each things. Also add instructions for first website with Django with demo.
Thank you!

**Comment by Ro-shni** [comment]
@Einsteinia11 sure, Thank you.

**Comment by Ro-shni** [comment]
Could you please check My new pull request on updated django page.

Also I've devloped content in numpy.html, if you could suggest some changes
that would be great. And then I'll make a pr for numpy.

**Comment by seabeePraveen** [comment]
hey i want to work on this issue and i also have a very good experience with django 
Thank You!

**Comment by rohit-rambade** [comment]
## Instructions

1. Yes you can start working on it by taking latest pull from main branch
2. Don't Forget to join onboarding group for knowledge flow and direct conversation and support from other developers link  [Curious Developers Community - Hacktoberfest ](https://chat.whatsapp.com/HNu3VaLPK3z3yt3JFUs7D7)
3. Please write the comments in your source code for development work and don't forget to attach the screenshots and video of the development work once it is done
4. Contributions more than 3 will be awarded a certificate of appreciation from curious ecosystem please mail to (admin@curiousdevelopers.in) for claiming certificate and please attach your issue number 
5. Respect Co Developers And Please Reachout to admins if any issues

Thanks & Regards
The Curious Team

**Comment by seabeePraveen** [comment]
Yeah thank you
and could u please specify more detail about the issue?

**Comment by rohit-rambade** [comment]
Make blog posts on Django library

**Comment by Ro-shni** [comment]
@thecuriousteam could you please verify pr #106 and suggest changes

**Comment by Ranjana993** [comment]
Hii , I would like to work on this issue .
